src/styles/blocks/body.css

```diff
@@ -0,0 +1,6 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 96px;
+  height: 96px;
```

Comment:
[checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md#css): Don't fix container size (if there is no such a requirement)